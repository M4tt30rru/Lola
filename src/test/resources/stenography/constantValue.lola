##Import "std.lola"
##Find
 ##MethodDeclaration(_dec) = ##Any(_value);
 ##replace ##(_dec) {return ##(_value);}

double sin(double x) = cos(x + pi/2);
