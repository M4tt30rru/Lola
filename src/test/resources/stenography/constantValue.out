double  sin (doublex) {return cos(x+pi/2);}
